WhaleSpotting/Migrations/20210804101051_editedSightingsDbModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WhaleSpotting.Migrations
+{
+    public partial class editedSightingsDbModel : Migration
```

Comment:
Could we regenerate this migration with a capital letter start. We use pascal case for migrations :)

Reply:
done :)